### ISSUE TITLE
Row and BelongsTo error

### ISSUE THREAD

**Issue by emiliogrv** [64robots/nova-fields#59]
Hi, with latest version a give an error with Row and BelongsTo when clicked on add row button in the create view and edit view

composer.json
```json
"64robots/nova-fields": "^0.6.1",
```

My own nova resource
```php
use R64\NovaFields\BelongsTo as CustomBelongsTo;
use R64\NovaFields\Number as CustomNumber;
use R64\NovaFields\Row;
use R64\NovaFields\Text as CustomText;
```
...

```php
Row::make('Lines', [
    CustomNumber::make('Cantidad', 'quantity')
        ->fieldClasses('w-full px-8 py-6')
        ->hideLabelInForms()
        ->min(1)
        ->rules(
            'required',
            'integer',
            'min:1'
        )
        ->sortable(),

    CustomText::make('Precio', 'price')
        ->fieldClasses('w-full px-8 py-6')
        ->hideLabelInForms()
        ->rules(
            'required',
            'string',
            'regex:/^\d+((,|\.)\d{1,2})?$/'
        )
        ->sortable(),

    /**
      * Relationships
      */
    CustomBelongsTo::make('Producto', 'product', Product::class)
        ->fieldClasses('w-full px-8 py-6')
        ->hideLabelInForms(),
])
->fieldClasses('w-full px-8 py-6')
->labelClasses('w-1/2 px-8 py-6'),
```

view
![1](https://user-images.githubusercontent.com/13983577/51069223-07042000-1601-11e9-8f88-883d6e53c4fc.png)

![2](https://user-images.githubusercontent.com/13983577/51069428-47659d00-1605-11e9-923c-bf4a6c791585.png)

**Comment by emiliogrv** [comment]
I think that the error is because your BelongsTo field search the associatable for the main resource, in my case issued-invoices and not for the Row Field that in my case is Lines.

Resume
IssuedInvoice have not Product relationship
Row is of Lines and this have Product relationship

That is I can think seeing the error in the console

**Comment by emiliogrv** [comment]
Hi, I leave a PR qith the soution to the issue: https://github.com/64robots/nova-fields/pull/60

Regards

**Comment by disitec** [comment]
Merged Pr. Fix available in upcoming release 0.6.2.

Thanks!
